pages/services/beta-confluent-kafka/2.1.2-4.0.0e-beta/kerberos/index.md

```diff
@@ -1,12 +1,15 @@
 ---
 layout: layout.pug
-navigationTitle: 
+navigationTitle:
 excerpt:
 title: Kerberos
 menuWeight: 22
-enterprise: false
```

Comment:
This diff looks a bit strange, but that's because the original file was deleted, and `git` is picking this up as a move and not a new addition.

The new file is the one added for the KAF-401 ticket.